package config;

// 路由策略
enum PolicyType{
	POLICY_ROUTER = 0;	
	POLICY_RANDOM = 1;			// 随机
	POLICY_MOD = 2;				// 取模
	POLICY_BROADCAST = 3;		// 广播
}

message Policy {
	optional uint32 svr_type = 1;
	optional uint32 policy = 2;
}

message RouterSvrConfig {
	optional string listen_ip = 1;
	optional int32 svr_inst_id = 2;
	repeated int32 legal_svr_list = 3;	// 合法服务列表
	repeated Policy policy_list = 4;
	optional int32 svr_zone = 5;
}